pontificia-universita-gregoriana-teologia.csl

```diff
@@ -0,0 +1,558 @@
+      <else-if type="article-newspaper" match="any">
+        <text variable="title" form="short" font-style="italic"/>
+      </else-if>
+      <!-- Gravaciones -->
+      <else-if type="broadcast musical_score song motion_picture" match="any">
+        <choose>
+          <if match="any" variable="collection-title">
+            <text variable="title" form="short" font-style="normal" quotes="true"/>
+          </if>
+          <else>
+            <text variable="title" form="short" font-style="italic"/>
+          </else>
+        </choose>
+      </else-if>
+      <else>
+        <text variable="title" form="short" quotes="true"/>
+      </else>
+    </choose>
+  </macro>
+  <!-- Título en bibliografía -->
+  <macro name="Titulo_Biblio">
+    <choose>
+      <!-- libros, tesis y manuscritos en cursiva, forma larga -->
+      <if type="book thesis manuscript" match="any">
+        <text variable="title" font-style="italic"/>
+        <choose>
+          <if type="thesis" match="any">
+            <text variable="genre" prefix=", "/>
+          </if>
+        </choose>
+      </if>
+      <!-- Articulos de periodico, especialmente AAS/ASS -->
+      <else-if type="article-newspaper" match="any">
+        <text variable="genre" suffix=" "/>
+        <text variable="title" form="short" font-style="italic" suffix=" "/>
+        <group prefix="(" suffix=")," delimiter=" ">
+          <date form="text" variable="issued">
+            <date-part name="day"/>
+            <date-part name="month"/>
+            <date-part name="year"/>
+          </date>
+        </group>
+      </else-if>
+      <!-- Gravaciones -->
+      <else-if type="broadcast musical_score song motion_picture" match="any">
+        <choose>
+          <if match="any" variable="collection-title">
+            <text variable="title" form="short" font-style="normal" quotes="true"/>
+          </if>
+          <else>
+            <text variable="title" form="short" font-style="italic"/>
+          </else>
+        </choose>
+      </else-if>
+      <else>
+        <text variable="title" quotes="true"/>
+      </else>
+    </choose>
+  </macro>
+  <!-- Volumen de la obra -->
+  <!-- Volumen en nota, precedido por 'vol.' y siempre en caracteres arábigos -->
+  <!-- Número de volumenes en bibliografía, seguido por 'vol.' y siempre en caracteres arábigos -->
+  <macro name="Volumen_Nota">
+    <choose>
+      <if match="any" is-numeric="volume">
+        <text term="volume" text-case="capitalize-first"/>
+        <text variable="volume" text-case="uppercase"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="Volumen_Biblio">
+    <choose>
+      <if match="any" is-numeric="number-of-volumes">
+        <number text-case="uppercase" variable="number-of-volumes" form="numeric"/>
+        <text term="volume" prefix=" "/>
+      </if>
+      <else-if match="any" is-numeric="volume">
+        <text term="volume" suffix=" " text-case="capitalize-first"/>
```

Comment:
```suggestion
  <macro name="Volumen_Nota">
    <choose>
      <if match="any" variable="volume">
        <text term="volume" text-case="capitalize-first" suffix=" "/>
        <text variable="volume" text-case="uppercase"/>
      </if>
    </choose>
  </macro>
  <macro name="Volumen_Biblio">
    <choose>
      <if match="any" is-numeric="number-of-volumes">
        <number text-case="uppercase" variable="number-of-volumes" form="numeric"/>
        <text term="volume" prefix=" "/>
      </if>
      <else-if match="any" variable="volume">
        <text term="volume" suffix=" " text-case="capitalize-first"/>
```

Volume may not be numeric, like in Norme Tipogrephiche p.28 line 759
Added also the space between vol. and number in footnote citation